src/styles/components/serviceProvider-card.scss

```
@@ -0,0 +1,31 @@
+.serviceProvider-card {
```

Hier auch lieber einheitlich kebab case: "service-provider-card"
```suggestion
.service-provider-card {
```